### ISSUE TITLE
Head & Neck segmentation

### ISSUE THREAD

**Issue by MattAWard** [wasserth/TotalSegmentator#58]
Hi are there any plans to facilitate H&N segmentation using TotalSegmentator? If not, is it possible to train our own model using marked-up CT data and their corresponding RTSTRUCT files?

**Comment by wasserth** [comment]
What exactly do you mean by head and neck segmentation? What structures would you like to have segmented?

**Comment by MattAWard** [comment]
This would be for segmentation relating to head and neck radiotherapy. Regions of interest typically include:

- Brain
- Brainstem
- Eyes
- Eye Lens
- Optic Chiasm
- Optic Nerves
- Cochlea
- Parotids
- Oral Cavity
- Mandible
- Spinal Canal/ Cord
- Oesophagus
- Thyroid Gland
- Brachial Plexus
- Carina

Thanks.

**Comment by MattAWard** [comment]
Any idea about this? As I said, an ability to train a model using our own anonymised CT data sets with these structures outlined would also be a great solution.

**Comment by wasserth** [comment]
Training a nnUNet should not be too difficult. You have to convert your data to nifti and follow the instructions of the nnU-Net Documentation (https://github.com/MIC-DKFZ/nnUNet). Having such a model would be great and then we could use this to help in annotation of the TotalSEgmentator dataset and then the TotalSegmentator could also do this.

**Comment by MattAWard** [comment]
That's great thanks @wasserth I'll look into training a model with some of our data and then we can take it from there.

**Comment by koegl** [comment]
> That's great thanks @wasserth I'll look into training a model with some of our data and then we can take it from there.

Have you trained a model for this task? I also have to segment H&N CTs and I wasn't happy with the outputs of TotalSegmentator

**Comment by wasserth** [comment]
In what regard were you unhappy with the TotalSegmentator results? Were there major missegmentations? Or rather small inaccuracies? Can you provide screenshots of the problems? Then we can try to improve the model.

**Comment by wasserth** [comment]
Thank you for sending these details. Are there other head/neck structures you are having making issues with?

**Comment by koegl** [comment]
For now, I only investigated the skull segmentation, as it is the simplest to validate visually.

I want to use those segmentations to evaluate registrations, so ideally, I would have segmentations of all structures, but I wanted to start with the bones as they should be easier.

I also noticed that not all structures described in the [head and neck paper](https://www.mdpi.com/2072-6694/16/2/415) are available in TotalSegmentator (e.g., mandible and cheekbone) - is there a reason for that?
